### PR TITLE
Implement waitUntilTimeout() function

### PR DESCRIPTION
**Pull request by eunikolsky**
This function allows to set an async spec timeout for the given block only in order not to change the global timeout, as `setAsyncSpecTimeout()` function does.

So it's easier and safer to write
```objc
waitUntilTimeout(1.0, ^(DoneCallback done) {
// …
});
```

than
```objc
setAsyncSpecTimeout(1.0);
waitUntil(^(DoneCallback done) {
// …
});
setAsyncSpecTimeout(10.0);
```

### CODE REVIEW

File: Specta/Specta/SpectaDSL.h [hunk 1]
```diff
@@ -48,6 +48,15 @@ OBJC_EXTERN void sharedExamples(NSString *name, void (^block)(NSDictionary *data
 #define      itBehavesLike(...) spt_itShouldBehaveLike_(@(__FILE__), __LINE__, __VA_ARGS__)
 
 OBJC_EXTERN void waitUntil(void (^block)(DoneCallback done));
+/**
+ * Runs the @c block and waits until the @c done block is called or the
```
Comment (orta) [inline Specta/Specta/SpectaDSL.h]: is `@c` special here, or are you just using it as a contraction of completion ( which is called done ) ?

Comment (eunikolsky) [inline Specta/Specta/SpectaDSL.h]: `@c` makes the next word rendered in monospace font, which highlights it as a parameter.

**Comment by orta** [comment]
Cool, this looks good to me then!

**Comment by orta** [comment]
This is released in 1.0.2

**Pull request merged**